onmt/opts.py

```diff
@@ -147,9 +147,9 @@ def model_opts(parser):
               help='Number of steps to advance graph encoder')
     # The ggnn uses src_vocab during training because the graph is built
     # using edge information which requires parsing the input sequence.
-    group.add('--src_vocab', '-src_vocab', default="",
-              help="Path to an existing source vocabulary. Format: "
-                   "one word per line.")
+    # group.add('--src_vocab', '-src_vocab', default="",
+    #           help="Path to an existing source vocabulary. Format: "
+    #                "one word per line.")
```

Comment:
?

Reply:
This conflicts with the src_vocab in preprocess_opts.